jetbrains-rider/resources/resharper/LambdaRunMarkers/CreateNew_dark.svg

```diff
@@ -0,0 +1 @@
+<svg ti:v='1' width='16' height='16' viewBox='0,0,16,16' xmlns='http://www.w3.org/2000/svg' xmlns:ti='urn:schemas-jetbrains-com:tisvg'><g><path d='M0,0L16,0L16,16L0,16Z' fill='#FFFFFF' opacity='0'/><path d='M9.977,3.613L11.466000000000001,2.143L10.308,1L8.12,3.17C8.0989999999999984,3.17,8.02,3.16,7.9999999999999991,3.16C7.9419999999999993,3.16,7.948999999999999,3.169,7.8909999999999991,3.1710000000000003L5.692,1L4.534,2.143L6.067,3.655C5.3364909117040122,4.029290841654487,4.72631480107167,4.6023710182881592,4.307,5.3079999999999989L2,5.3079999999999989L2,6.923L3.753,6.923C3.72300187244563,7.1012843897325979,3.7046297608820238,7.2813310830559388,3.6980000000000013,7.462L3.6980000000000013,8.538L2,8.538L2,10.154L3.7,10.154L3.7,10.692C3.7048005678728435,11.05626599221125,3.7572462375444964,11.418342827059778,3.8560000000000016,11.769L2,11.769L2,13.385L4.687,13.385C5.2300321490649608,14.059331369401214,5.96295264087832,14.555628763076555,6.790689805050051,14.80951436275792C7.6184269692217823,15.063399962439284,8.50357303077822,15.063399962439284,9.3313101949499515,14.80951436275792C10.159047359121681,14.555628763076555,10.891967850935039,14.059331369401214,11.434999999999999,13.385L14,13.385L14,11.769L12.269,11.769C12.36726448907206,11.418239066076532,12.419700826147956,11.05622666203333,12.425,10.692L12.425,10.154L14,10.154L14,8.538L12.425,8.538L12.425,7.462C12.418039479361761,7.2813092475768668,12.39933331444052,7.10126241020991,12.369000000000003,6.923L14,6.923L14,5.308L11.815,5.308C11.381178971736929,4.5756159850921456,10.7420490794503,3.9862116502576312,9.977,3.6129999999999991ZM9.977,3.613M9.8,11.5L6.2,11.5L6.2,10L9.8,10ZM9.8,11.5M9.8,8.5L6.2,8.5L6.2,7L9.8,7ZM9.8,8.5' fill='#499C54'/></g></svg>
```

Comment:
Do these need to be the real icons, given they get patched through RiderAwsIconsPatcher or can they just be empty svg files?

Reply:
This could be replaced with an empty svg files. The only reason I took into account when left them as is - to get a backup image when icon patcher fail by any reason. I'll update with empty icons then.